docs/source/rclone.rst

```diff
@@ -18,6 +18,29 @@ Services like our JupyterHub and the FARM already have rclone installed. To chec
 
     **FARM rlcone**: two different versions of ``rclone`` live on the farm. Make sure you're using the one from ``/software/rclone/rclone`` because that's the latest.
 
+Add rclone to your FARM PATH
+----------------------------
+
+You add can ``/software/rclone`` to your path to have the latest rclone version avaiable on the FARM ready to use. ``/softwarre/rclone`` is a symlink to the latest version of rclone installed on the farm. In order to do this you need to edit your ``~/.bashrc`` file and add:
+
+  .. code-block:: bash
+    
+    export PATH="/software/rclone:${PATH}"
+
+Then ``source ~/.bashrc`` or logout/re-login to the FARM to have the changes applied.
+
+
+FARM module load
```

Comment:
Updated the module with the latest verssion (1.60.1). This IDS option is not the best since they don't have a `latest` module avaiable. However, there's a symlink between `/software/rclone` --> latest installed and I've addeed the above section  ("Add rclone to your FARM PATH") to cover that.